inst/rmd/h/anchor-sections/anchor-sections-hash.css

```diff
@@ -0,0 +1,2 @@
+/* Styles for section anchors */
+a.anchor-section::before {content: '#';font-size: 80%;}
```

Comment:
We can remove this file if we accept:
https://github.com/rstudio/rmarkdown/pull/1964#discussion_r533428044

Reply:
Ok. but there is the `font-size` that I added only for the dash style. 
It would apply also on other style if we put it in the default css file right ?

Reply:
Ah, yes. Let's keep the current manner then. Alhough I see the other CSS defines `font-size` by themselves if needed, this will be a better choice that reduces unexpected conflicts with the additional styles in the future.
https://github.com/rstudio/rmarkdown/pull/1964/files#diff-a354a361dd9b24096377c197932c131ed2d45352ebd40d80cff96855829b278aR4

Reply:
yes there is non when I added the icon. I think I'll keep the 3 files
* Common rules in `anchor-sections.css`
* Specific rule for style argument in `anchor-sections-<style>.css`